src/config/field-overrides/program-notification-template/RelativeScheduledDays.js

```diff
@@ -39,18 +42,18 @@ function RelativeScheduledDays({ onChangeBeforeAfter, beforeOrAfter, onChangeDay
 
     /* Because of bad alignment of material ui textfield and selectfield, the compoents becomes skewed when
        using the hide method EditModelForm.component (setting the display to none/block). The component 
-       must instead chose to either use display:none from style or its defined display:flex. 
+       must instead chose to return on display:'none'
     */
-    const wrapStyle = (style && (style.display === 'none'))
-        ? { ...relativeScheduledDaysStyle, ...style }
-        : relativeScheduledDaysStyle;
+    if (style && (style.display === 'none')) {
+        return null;
+    }
 
     return (
-        <div style={wrapStyle}>
+        <div style={relativeScheduledDaysStyle}>
             <span>{t('send_notification')}</span>
             <TextField
                 hintText={t('number_of_days')}
-                value={Math.abs(value || 0)}
+                value={Math.abs(value)}
```

Comment:
Should probably leave the default? Or you will get NaN if you switch the "before" or "after" with no value.